src/app/manage-users/manage-user-creation/manage-user-creation.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { UsersService } from 'src/app/shared/users.service';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-manage-user-creation',
+  templateUrl: './manage-user-creation.component.html',
+  styleUrls: ['./manage-user-creation.component.css']
+})
+export class ManageUserCreationComponent implements OnInit {
+
+  @Output() listChanged: EventEmitter<any> = new EventEmitter();
+
+  constructor(private usersService:UsersService) { }
+
+  ngOnInit(): void {
+  }
+
+  createNewUser(mainForm) {
+    this.usersService.createUser(
+      mainForm.value
+    ).subscribe(
+      (result) => {
+        console.log('Result: Create User API - ', result);
+        mainForm.reset();
+      },
+      (error) => {
+        console.log('Error: Create User API - ', error);
+      },
+      () => {
+        console.log('Complete: Create User API');
+      }
```

Comment:
These blocks can be removed if you are not using them. Also when a user is created successfully, the ideal behavior is to navigate the user to users-list page where we can see the newly created user

Reply:
did this